libsql-server/src/http/user/extract.rs

```diff
@@ -26,19 +29,21 @@ impl FromRequestParts<AppState> for RequestContext {
             .with(namespace.clone(), |ns| ns.jwt_key())
             .await??;
 
-        let auth_header = parts.headers.get(hyper::header::AUTHORIZATION);
+        let context = parts.headers
+        .get(hyper::header::AUTHORIZATION).context("auth header not found") // todo this context is swallowed for now, gotta fix that but not with panicking
```

Comment:
Agreed, on the removing the panic 👍 I think moving to thiserror based on my comment above would be good here.